mappings/com/mojang/blaze3d/platform/FramebufferInfo.mapping

```diff
@@ -1,4 +1,17 @@
-CLASS net/minecraft/class_4492 com/mojang/blaze3d/platform/FramebufferInfo
-	FIELD field_20457 target I
-	FIELD field_20458 renderBufferTarget I
-	FIELD field_20460 attachment I
+CLASS net/minecraft/class_4492 com/mojang/blaze3d/platform/FrameBufferInfo
```

Comment:
leave it `Framebuffer`

Reply:
Yeah, id leave it, the file didnt get renamed either way.

Reply:
I'll change it back, though I still believe Frame Buffer should be two words.

There is an issue here that git doesn't record filename changes when the only change is a case difference though.